src/Volume.ts

```diff
@@ -129,6 +129,9 @@ interface VolumeDataObserver {
 export default class Volume {
   public imageInfo: ImageInfo;
   public loadSpec: LoadSpec;
+  public loader?: IVolumeLoader;
```

Comment:
what does it mean if there's no loader?

Reply:
My thought here was to preserve the ability to just directly construct a volume and add data to it via methods, in which case any side effects from the whole `updateRequiredData` system would get captured by the null check on this loader and just not happen.